analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/PublicMethodArgumentsShouldBeCheckedForNullBase.cs

```diff
@@ -67,4 +83,10 @@ private static IOperation NullDereferenceCandidate(IOperation operation)
         };
         return candidate?.UnwrapConversion();
     }
+
+    private static IOperation AssignmentTarget(IOperation operation) =>
+        // Missing operation types: DeconstructionAssignment, CoalesceAssignment, CompoundAssignment, ThrowExpression
```

Comment:
I think "assignment with flow capture" is missing here, right?
```suggestion
        // Missing operation types: DeconstructionAssignment, CoalesceAssignment, CompoundAssignment, ThrowExpression, assignment with flow capture
```